code/modules/lighting/lighting_atom.dm

```diff
@@ -46,21 +46,13 @@
 		T.reconsider_lights()
 	return ..()
 
-/atom/movable/Move()
-	var/turf/old_loc = loc
+/atom/Entered(atom/movable/obj, atom/prev_loc)
```

Comment:
Assigning to `loc` directly anywhere will not update lighting, so it may be a good idea to get used to using `forceMove()`